src/handlers.rs

```diff
@@ -265,7 +265,7 @@ fn handle_code_cache(d: *const isolate, base: &msg::Base) -> Box<Op> {
   }()))
 }
 
-fn handle_set_env(d: *const isolate, base: &msg::Base) -> Box<Op> {
+fn handle_set_env(d: *const isolate_ptr, base: &msg::Base) -> Box<Op> {
```

Comment:
I meant the parameter name, not the type.

Reply:
sure, I got it.